gmso/tests/files/ff-example0.xml

```diff
@@ -6,14 +6,14 @@
         <ParametersUnitDef parameter="A" unit="kcal/mol"/>
         <ParametersUnitDef parameter="B" unit="nm"/>
         <ParametersUnitDef parameter="C" unit="kcal/mol*nm**6"/>
-        <AtomType name="Ar" element="Ar" charge="0.0" mass="39.948" definition="Ar" description="Argon atom">
+        <AtomType name="Ar" element="Ar" charge="0.0" atomclass="Ar" mass="39.948" definition="Ar" description="Argon atom">
             <Parameters>
                 <Parameter name="A" value="0.1"/>
                 <Parameter name="B" value="4.0"/>
                 <Parameter name="C" value="0.5"/>
             </Parameters>
         </AtomType>
-        <AtomType name="Xe" element="Xe" charge="0.0" mass="131.293" definition="Xe" description="Xenon atom">
+        <AtomType name="Xe" element="Xe" atomclass="Xe" charge="0.0" mass="131.293" definition="Xe" description="Xenon atom">
```

Comment:
per discussion in call, we should change atomclass to class, as well as the kwarg in `AtomType` too

Reply:
See #445